lib/scala/syntax/specialization/shared/src/test/scala/org/enso/syntax/text/DocParserTests.scala

```diff
@@ -752,14 +754,12 @@ class DocParserTests extends AnyFlatSpec with Matchers {
 
   """
     | - bar
-    | baz
+    |   baz
```

Comment:
You need more tests here. What happens if you have block text after a list (sep by a blank line or indentation). What happens if you have multiple lists separated by a blank line? Does this work inside examples, important blocks, etc?